cobra/test/test_io/test_pickle.py

```diff
@@ -38,7 +38,7 @@ def test_write_pickle(tmpdir, mini_model, dump_function):
         pytest.skip()
 
     output_file = tmpdir.join("mini.pickle")
-    with open(str(output_file), "wb") as outfile:
+    with open(output_file.strpath, "wb") as outfile:
```

Comment:
I actually find the previous version using `__str__` quite idiomatic. Easy to grasp what's meant.

Reply:
If you are not merging yet, I'll push another commit addressing this.